packages/notification-center/package.json

```diff
@@ -7,9 +7,13 @@
     "start": "npm run build:watch",
     "prebuild": "rimraf dist",
     "lint": "eslint --ext .ts,.tsx src",
-    "build": "rollup -c && npm run build:types",
+    "build": "npm run build:cjs && npm run build:esm && npm run build:umd && npm run build:types",
+    "build:cjs": "cross-env node_modules/.bin/tsc -p tsconfig.json",
+    "build:esm": "cross-env node_modules/.bin/tsc -p tsconfig.esm.json",
+    "build:esm:watch": "cross-env node_modules/.bin/tsc -p tsconfig.esm.json -w --preserveWatchOutput",
+    "build:umd": "webpack --config webpack.config.js",
     "build:types": "tsc --declaration --emitDeclarationOnly --declarationMap --declarationDir dist/types -p tsconfig.json",
-    "build:watch": "rollup -c -w",
+    "build:watch": "npm run build:esm:watch",
```

Comment:
That latest Axios version broke the UMD build of the `notification-center` package. 
First of all, Rollup was bundling the Axios for node env, which required to polyfill NodeJS native modules like `http, utils, ...` etc. 
The second thing was that the Rollup NodeJS polyfills do not support everything that Axios is using right now in Axios for node.
I wasn't able to figure out why it was trying to bundle the Axios for node, but I was sure that it was the issue between rollup plugins.

After a long fight, I decided to give it a try to Webpack, and in a few minutes, I had it working like a charm.
Then CJS and ESM modules needed to be built with TSC, but we were importing some images in the components, so I had to fix them too, like to make them React SVG components.